src/test/llmnode_test.cpp

```diff
@@ -1601,23 +1606,23 @@ TEST_F(LLMFlowHttpTest, inferCompletionsStreamClientDisconnectedImmediately) {
         }
     )";
 
-    EXPECT_CALL(writer, IsDisconnected())
+    EXPECT_CALL(*writer, IsDisconnected())
         .WillOnce(::testing::Return(true));
 
     std::atomic<int> i = 0;
-    EXPECT_CALL(writer, PartialReplyEnd()).Times(1);
-    EXPECT_CALL(writer, PartialReplyWithStatus(::testing::_, ::testing::_)).WillOnce([this, &i](std::string partialResponse, tensorflow::serving::net_http::HTTPStatusCode code) {
+    EXPECT_CALL(*writer, PartialReplyEnd()).Times(1);
+    EXPECT_CALL(*writer, PartialReplyWithStatus(::testing::_, ::testing::_)).WillOnce([this, &i](std::string partialResponse, ovms::HTTPStatusCode code) {
         i++;
         ASSERT_EQ(partialResponse, "{\"error\":\"Mediapipe execution failed. MP status - CANCELLED: CalculatorGraph::Run() failed in Run: \\nCalculator::Process() for node \\\"llmNode1\\\" failed: \"}");
         rapidjson::Document d;
         rapidjson::ParseResult ok = d.Parse(partialResponse.c_str());
         ASSERT_EQ(ok.Code(), 0);
-        ASSERT_EQ(code, tensorflow::serving::net_http::HTTPStatusCode::BAD_REQUEST);
+        // TODO
```

Comment:
Leave comment what we need to resolve this TODO

Reply:
done